packages/lesswrong/components/ea-forum/TargetedJobAd.tsx

```diff
@@ -176,9 +176,30 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 })
 
+type JobAdDataType = {
+  [key: string]: {
```

Comment:
Nit: It might be cleaner to define this type without the `[key: string]` and then to make `JOB_AD_DATA` a `Record<string, JobAdDataType>`. Super low-importance though.

Another nit: perhaps just `JobAdData` is nicer than `JobAdDataType`(?), but maybe other people have other opinions - I couldn't see anything about putting `Type` in a typename in the coding conventions doc, but off the top of my head I can't think of many other places where we do this.